builder/build.go

```diff
@@ -450,19 +450,19 @@ func (p *Package) BuildXML(notif PidNotifier, pman *EopkgManager, overlay *Overl
 	}
 
 	// Now build the package, ignore-sandbox in case someone is stupid
-	// and activates it in eopkg.conf..
+	// and activates it in eopkg.conf...
 	cmd := eopkgCommand(fmt.Sprintf("eopkg build --ignore-sandbox --yes-all -O %s %s", wdir, xmlFile))
 
-	log.Infof("Now starting build of package %s\n", p.Name)
+	slog.Info(fmt.Sprintf("Now starting build of package %s\n", p.Name))
```

Comment:
Out of curiosity, why is this one taking a pre-formatted string? Is it because it's an info line?